File: src/beginner/day08/Question04.java
package beginner.day08;

public class Question04 {

    public static class Solution {

        public int solution(int n) {
            int answer = 0;

            for (int i = 1; i <= n; i++) {
                if (n % i == 0) {
                    answer++;
                }
            }
            
            return answer;
        }
    }

    public static void main(String[] args) {

        Solution solution = new Solution();

        System.out.println(solution.solution(20));
        System.out.println(solution.solution(100));
    }
}

File: src/beginner/day19/Question04.kt
package beginner.day19

class Question04{

    fun solution(array: IntArray, height: Int): Int {
        var answer: Int = 0

        for (i in array) {
            if (height < i) answer++
        }

        return answer
    }
}

fun main() {

    val solution = Question04()

    val array1 = intArrayOf(149, 180, 192, 170)
    val array2 = intArrayOf(180, 120, 140)

    println(solution.solution(array1, 167))
    println(solution.solution(array2, 190))
}